.azure-pipelines/scripts/sap_hana/linux/50_set_up_sysctl.sh

```diff
@@ -2,14 +2,10 @@
 
 set -ex
 
-# 6GB is proven to be enough for SAP HANA to run.
-# The default number of memory addresses is 65536, i.e. 512MB (Linux 64-bit).
-# => To get 6GB, we multiply that amount by 12.
-sudo sysctl -w vm.max_map_count=$(expr 16 \* 65536)
-
 # Recommended settings on https://developers.sap.com/tutorials/hxe-ua-install-using-docker.html
 sudo sysctl -w fs.file-max=20000000
 sudo sysctl -w fs.aio-max-nr=262144
+sudo sysctl -w  vm.max_map_count=135217728
```

Comment:
Where did you get this recommendation from?

Reply:
from the link that is 3 lines above

Reply:
Great. I did not see it